### ISSUE TITLE
`Attachment`: Editing lecture-attachments is not possible in Safari due to an overlay

### ISSUE THREAD

**Issue by DominikRemo** [ls1intum/Artemis#6624]
### Describe the bug

When trying to edit an attachment for a lecture (not the lecture units but the lecture attachment), the entire screen is filled with the "is currently being edited..." overlay.

### To Reproduce

Prerequisites:
1 Instructor
1 Student

1. Create a lecture in a course
2. Navigate to the "Attachments" of this lecture (not the attachment units!)
3. Create an attachment with a new file
4. Edit the attachment
Notice how the form fields are blocked by the overlay.

### Expected behavior

The overlay should only be above the table, not the entire screen.

### Screenshots

Chrome:
<img width="1200" src="https://github.com/ls1intum/Artemis/assets/47261058/e35238df-642f-49e0-a77f-4b10d86fd480">

Safari:
<img width="1200" src="https://github.com/ls1intum/Artemis/assets/47261058/1d49afae-e6bd-45a5-8c94-4153dc08858b">

### Which version of Artemis are you seeing the problem on?

6.1.8

### What browsers are you seeing the problem on?

Safari

**Comment by maximiliansoelch** [comment]
Duplicate of #6195
